nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```diff
@@ -50,6 +50,15 @@
           '';
         };
 
+        suppressNvidiaDriverAssertion = lib.mkOption {
+          default = false;
```

Comment:
Could have been `mkEnableOption ''...''`

Reply:
Oh, right. Didn't realize... I'll take advantage when creating another PR that involves nvidia-ctk to update it.